README.md

```diff
@@ -167,6 +167,7 @@ semantic-release
 
 These options are currently available:
 - `branch`: The branch on which releases should happen. Default: `'master'`
+- `repositoryUrl`: The fit repository URL. Default: git origin url or `repository` property in `package.json`
```

Comment:
```diff
- The fit repository URL
+ the git repository URL
```

Reply:
What format can the `git` repository URL have? Maybe we could add a few examples or link to the package we use to parse the URL?

Reply:
I fixed the typo and added a note regarding the supported format.
We do not support the shorthand format (for example `npm/npm`) because [release-notes-generator](https://github.com/semantic-release/release-notes-generator) needs to determine the hostname to create links in the release notes. With the shorthand format we don't have a way to figure out if the hostname is `github.com`, `gitlab.com` or any custom GHE domain.
I will check if we can generate release notes with relative URLs. If it's the case I will create a PR on `release-notes-generator` to handle that and remove the restriction in the `semantic-release` doc.